src/terminal/parser/IStateMachineEngine.hpp

```diff
@@ -46,9 +46,7 @@ namespace Microsoft::Console::VirtualTerminal
 
         virtual bool ActionIgnore() = 0;
 
-        virtual bool ActionOscDispatch(const wchar_t wch,
-                                       const size_t parameter,
-                                       const std::wstring_view string) = 0;
+        virtual bool ActionOscDispatch(const size_t parameter, const std::wstring_view string) = 0;

```

Comment:
I dropped the `wch` parameter from this method because it's never actually used for anything. It just indicated whether the sequence was terminated with a `BEL` or an `ST` backslash, but that's of no relevance to anyone.